scripts/retrieve_databundle_light.py

```diff
@@ -95,6 +95,7 @@
     create_logger,
     progress_retrieve,
 )
+from build_demand_profiles import log_load_paths_gegis
```

Comment:
Mmm on second thought, maybe it is not well placed here but rather in build_demand profiles.
By looking at the function build_demand_profiles in build_demand rule, there is:
`logger.info(f"Merging demand data from paths {load_paths} into the load data frame")`
that should already explain the loaded load paths, so probably this should be enough.

As the CI will be working, we can check it here online.
So here probably we could restore the previous behavior.